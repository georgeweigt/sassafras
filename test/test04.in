data wine ;
input color $ x1 x2 x3 x4 x5 x6 x7 x8 x9 x10 x11 y ;
infile "wine.txt" ;
proc means n mean clm ;
var y ;
class color ;
