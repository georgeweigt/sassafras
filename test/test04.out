     COLOR     Variable        N      Mean     95% CLM MIN     95% CLM MAX
     red       Y            1599     5.636           5.596           5.676
     white     Y            4898     5.878           5.853           5.903

